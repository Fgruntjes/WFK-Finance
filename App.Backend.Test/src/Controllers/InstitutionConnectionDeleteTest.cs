namespace App.Backend.Test.Controllers;

public class InstitutionConnectionDeleteTest : IClassFixture<InstitutionConnectionDeleteFixture>
{
	private readonly InstitutionConnectionDeleteFixture _fixture;

	public InstitutionConnectionDeleteTest(InstitutionConnectionDeleteFixture fixture)
	{
		_fixture = fixture;
	}

	[Fact]
	public async Task Success()
	{
		var result = await _fixture.ExecuteQuery(new
		{
			ConnectionIds = new List<Guid> { _fixture.InstitutionConnectionId }
		});
		result.MatchSnapshot();
	}

	[Fact]
	public async Task MissingConnection()
	{
		var result = await _fixture.ExecuteQuery(new
		{
			ConnectionIds = new List<Guid> { new("5dcb861b-b879-427e-ad47-4c4eade20813") }
		});
		result.MatchSnapshot();
	}
}