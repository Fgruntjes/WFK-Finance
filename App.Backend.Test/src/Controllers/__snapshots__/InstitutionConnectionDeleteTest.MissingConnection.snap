{
  "data": {
    "institutionConnection": {
      "delete": 0
    }
  }
}